Chutzpah/TestRunner.cs

```diff
@@ -684,8 +684,9 @@ private static string BuildRunnerArgs(TestOptions options, TestContext context,
             string runnerArgs;
             var testModeStr = testExecutionMode.ToString().ToLowerInvariant();
             var timeout = context.TestFileSettings.TestFileTimeout ?? options.TestFileTimeoutMilliseconds ?? Constants.DefaultTestFileTimeout;
-
-            runnerArgs = string.Format("--ignore-ssl-errors=true --proxy-type=none --ssl-protocol=any \"{0}\" {1} {2} {3} {4} {5}",
+            var proxySetting = string.IsNullOrEmpty(options.Proxy) ? (string.IsNullOrEmpty(context.TestFileSettings.Proxy)?"--proxy-type=none": string.Format("--proxy={0}", context.TestFileSettings.Proxy)) : string.Format("--proxy={0}",options.Proxy);

```

Comment:
okay, if the proxy setting is set in both json file and passed from command line arguments, should the command line override the json setting or another way around?